xbmc/filesystem/PluginFile.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "PluginFile.h"
+#include "URL.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, **with an empty line clearly separating sections.**

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers